src/coreclr/vm/prestub.cpp

```diff
@@ -2618,60 +2644,76 @@ extern "C" PCODE STDCALL PreStubWorker(TransitionBlock* pTransitionBlock, Method
 
         pPFrame->Push(CURRENT_THREAD);
 
-        INSTALL_MANAGED_EXCEPTION_DISPATCHER;
-        INSTALL_UNWIND_AND_CONTINUE_HANDLER;
+        EX_TRY
+        {
+            bool propagateExceptionToNativeCode = IsCallDescrWorkerInternalReturnAddress(pTransitionBlock->m_ReturnAddress);
```

Comment:
Does this problem exist in other places that use `INSTALL_MANAGED_EXCEPTION_DISPATCHER_EX

For example, can ExternalMethodFixupWorker or VSD_ResolveWorker end up appear to be called from CallDescrWorker?

Reply:
That's a good point. I've checked and verified that both of the cases you've mentioned can be called via `CallDescrWorker`, so both suffer from the same issue in case there is an exception stemming from the resolution of the target. I think there are no other cases than the ones you've mentioned.